addon/services/hover-intent.js

```diff
@@ -0,0 +1,126 @@
+import Ember from 'ember';
```

Comment:
Not sure I understand, prefix what with `-` ? 
```
import -Ember from 'ember';
``` 
and 
```
import Ember from '-ember';
```
both fail

prefix the actual filename?